.github/workflows/Build.yaml

```diff
@@ -173,22 +207,30 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
 
-      - name: Build projects before running emulator
+      - name: Accept Android licenses
+        run: yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager --licenses || true
+
+      - name: Build AndroidTest apps
         run: ./gradlew packageDemoDebug packageDemoDebugAndroidTest
 
-      - name: Run instrumentation tests
-        uses: reactivecircus/android-emulator-runner@v2
+      - name: Run instrumented tests with GMD
+        run: ./gradlew ciDemoDebugAndroidTest --no-parallel --max-workers=1
```

Comment:
Also, we have 3 parallel flags enabled in this project:

<img width="218" alt="Screenshot 2024-02-09 at 01 43 19" src="https://github.com/android/nowinandroid/assets/15968667/933e58b9-b85a-4043-b737-0d67d5d76c6e">

I don't think disabling it would make sense here. WDYT?